README.md

```
@@ -1 +1,284 @@
+5. **Explore Psychology**: How does knowing your opponent affect decisions?
+
+### 7. Start the Frontend
+
+```bash
+pnpm --filter frontend dev
+```
+
+### 9. Test Accounts & Funding
+
+**Deployer Account:**
+* Address: `0x3f1Eae7D46d88F08fc2F8ed27FCb2AB183EB2d0E`
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: dash; Actual: asterisk

(MD004, ul-style)